### PR TITLE
ci: fix gentx action task with missing env variables

### PR DESCRIPTION
**Pull request by prajjwol**

### CODE REVIEW

File: .github/workflows/gentx_format.yml [hunk 1]
```diff
@@ -15,3 +14,13 @@ jobs:
...
           bash -x ./scripts/validate-gentxs.sh
+        env:
+          DAEMON: evmosd
+          GOPATH: $HOME/go
+          CHAIN_ID: evmos_9000-2
+          DENOM: aphoton
+          GH_URL: https://github.com/tharsis/evmos
+          BINARY_VERSION: v0.2.0
+          GO_VERSION: 1.17
+          PRELAUNCH_GENESIS_URL: https://raw.githubusercontent.com/tharsis/testnets/main/olympus_mons/genesis.json
```
Comment (fedekunze) [inline .github/workflows/gentx_format.yml:25]: we need to create this one I assume

File: scripts/validate-gentxs.sh [hunk 2]
```diff
@@ -19,31 +19,31 @@ echo
...
-  exit 0
+  exit 1
 fi
 if [[ -z "${GENTXS_DIR}" ]]; then
   echo "GENTXS_DIR in not set, required."
-  exit 0
+  exit 1
 fi
 
 command_exists () {
```
Comment (fedekunze) [inline scripts/validate-gentxs.sh:49]: we need to update L61 too

File: .github/workflows/gentx_format.yml (hunk 1 above)
Comment (prajjwol) [reply .github/workflows/gentx_format.yml:28]: is that why it's failing? https://github.com/tharsis/testnets/runs/4242167112?check_suite_focus=true

File: scripts/validate-gentxs.sh [hunk 3]
```diff
@@ -12,71 +12,32 @@ CHAIN_ID= evmos_9000-2
...
 if [[ -z "${GENTXS_DIR}" ]]; then
   echo "GENTXS_DIR in not set, required."
-  exit 0
-fi
-
-command_exists () {
-    type "$1" &> /dev/null ;
-}
-
-if command_exists go ; then
```
Comment (prajjwol) [inline scripts/validate-gentxs.sh:53]: adding `setup-go` action instead!

**Review by jolube — APPROVED** [review]

**Comment by giansalex** [comment]
@prajjwol missing add $GENACC  ? `$DAEMON add-genesis-account $GENACC 1000000000000000$DENOM --home $DAEMON_HOME`
https://github.com/tharsis/testnets/blob/5b48d3132298bb9bb30689e80fddbddf78459c89/scripts/validate-gentxs.sh#L80

**Pull request merged**